ui/src/alerting/components/ChecksColumn.tsx

```diff
@@ -25,19 +26,18 @@ const ChecksColumn: FunctionComponent<Props> = ({
     router.push(`/orgs/${orgID}/alerting/checks/new`)
   }
   return (
-    <>
-      <AlertsColumnHeader title="Checks" onCreate={handleClick} />
+    <AlertsColumnHeader title="Checks" onCreate={handleClick}>
       <CheckCards checks={checks} />
-    </>
+    </AlertsColumnHeader>
   )
 }
 
 const mstp = (state: AppState) => {
-  const {
-    checks: {list: checks},
-  } = state
+  // const {
+  //   checks: {list: checks},
+  // } = state
 
-  return {checks}
+  return {checks: [check1, check2]}
```

Comment:
ah, maybe we don't need to check this in?

Reply:
Wooooooops, fixing now!